depthcharge/components/encoders.py

```diff
@@ -28,14 +28,14 @@ def __init__(self, dim_model, min_wavelength=0.001, max_wavelength=10000):
             base = min_wavelength / (2 * np.pi)
             scale = max_wavelength / min_wavelength
         else:
-            base = 1
-            scale = max_wavelength / (2 * np.pi)
+            base = 1 / (2 * np.pi)
+            scale = max_wavelength
 
-        sin_term = base * scale ** (
-            torch.arange(0, n_sin).float() / (n_sin - 1)
+        sin_term = base * (
+            scale ** (torch.arange(0, n_sin).float() / (n_sin - 1))
         )
-        cos_term = base * scale ** (
-            torch.arange(0, n_cos).float() / (n_cos - 1)
+        cos_term = base * (
+            scale ** (torch.arange(0, n_cos).float() / (n_cos - 1))
```

Comment:
**question:** Are we missing the multiplication by 2 and the `- 1` in the exponent of the `scale` here?

Reply:
This is similar to answer above - I'll make the change!